test.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+
+using namespace std;
+
+int main() {
+    int n = 10;
+    int count = 0;
+
+    for(int i = 1; i <= n/2; i*=2){
+	for(int j = i; j < 4*i; j*=2) {
+	    count++;
+	}
+    }
+
+    cout << count << endl;
+}
```

Comment:
Umm what's this?

Reply:
Oops this was something I was using to test something totally unrelated I did not realize I saved it in the penn mobile repository LOL